homeassistant/components/withings/common.py

```diff
@@ -85,13 +96,11 @@ class WithingsAttribute:
     """Immutable class for describing withings sensor data."""
 
     measurement: Measurement
-    measute_type: Enum
+    measure_type: MeasureType | NotifyAppli | GetSleepSummaryField
     friendly_name: str
-    unit_of_measurement: str
-    icon: str | None
     platform: str
-    enabled_by_default: bool
     update_type: UpdateType
+    entity_description: EntityDescription
```

Comment:
This is now becoming a weird mix between how it used to be done, vs how we do it nowadays.

Entity description can be extended to hold all this information, instead of having a custom attribute type that also includes the entity description.

I suggest to avoid doing this, and instead migrate it to use entity description in the way we do everywhere within Home Assistant.

../Frenck

Reply:
You're right, but I thought this would be a step in the right direction and avoid too big a change.
I'll adjust...